openpype/settings/entities/schemas/projects_schema/schemas/schema_global_tools.json

```diff
@@ -408,6 +408,60 @@
                             }
                         ]
                     }
+                },
+                {
+                    "type": "list",
+                    "key": "transient_dir_profiles",
+                    "label": "Transient Dir Profiles",
+                    "use_label_wrap": true,
+                    "docstring": "Profiles to specify special location and persistence for staging dir. Could be used in Creators and Publish phase!",
+                    "object_type": {
+                        "type": "dict",
+                        "children": [
+                            {
+                                "key": "hosts",
+                                "label": "Host names",
+                                "type": "hosts-enum",
+                                "multiselection": true
+                            },
+                            {
+                                "key": "task_types",
+                                "label": "Task types",
+                                "type": "task-types-enum"
+                            },
+                            {
+                                "key": "task_names",
+                                "label": "Task names",
+                                "type": "list",
+                                "object_type": "text"
+                            },
+                            {
+                                "key": "families",
+                                "label": "Families",
+                                "type": "list",
+                                "object_type": "text"
+                            },
+                            {
+                                "key": "subsets",
+                                "label": "Subset names",
+                                "type": "list",
+                                "object_type": "text"
+                            },
+                            {
+                                "type": "separator"
+                            },
+                            {
+                                "key": "use_transient_dir",
+                                "label": "Use Transient Folder",
+                                "type": "boolean"
+                            },
```

Comment:
If we have a profile set, why would this ever be disabled?

Reply:
I think this is just question of preference, this way it might be easier to hot wire it and deactivate for debugging/production issue.

Reply:
I'd say if it's a global enable/disable for the entry in the list maybe it's better to name the key `enabled` like many other parts of settings do. And if somehow possible it'd be extra nice if it's like a header checkbox slider on this entry like seen in other areas of the UI.

Reply:
Here is some example from Flame and if it is really just about hot wiring, it might make more sense to make it this way
![image](https://user-images.githubusercontent.com/40640033/225697715-d8e56cd8-d5c4-4d48-af3c-410e28b75fdf.png)